eras/allegra/impl/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Version history for `cardano-ledger-allegra`
 
+## 1.2.0.0
+
+* Introduction of `TxCert` and `EraTxCert`
+* Add `EraTxCert` and `ShelleyEraTxCert` instances to `AllegraEra`
+
```

Comment:
@lehins I realize now that by releasing a bunch of ledger packages on CHaPs that were not from master, the changelogs are now a bit wrong. For example, the `ShelleyEraDCert` addition mentioned on line 14 of this changelog should go in `1.2.0.0` now, since it was not added to `1.1.1.0`.

After we merge your PR here, I can clean it all up.

Reply:
Yeah, that's exactly what I was trying to fixup in this PR. Guess I missed some. I did fix what mentioned here, but if you can double check all changelog files after this PR is merged that will be great